Version 1.0 (Rewrite)

noaione opened the pull request:
Adding more website support:
- [x] GYAO
- [x] Aniplus
~~Bilibili~~ (Will be finished after version 1.0)
~~Ponimu~~ (Will be finished after version 1.0)

General Rewrite
- [x] Downloader
- [x] Parser
- [x] Multi-website support
- [x] Commands 

Testing:
- [x] Test new parser
- [x] Test download
- [x] Test everything else

EDIT:
It's kinda working now, try it with: `pip install git+https://github.com/noaione/yuu.git@rewrite`

Commit:
Rewrite Progress 1

- Rewrite command to utilize multiple parser
- Rewrite abematv parser to class
- Start making GYAO parser
- Simplify downloader
- Modify save folder to:
	- Windows: AppData/Local/yuu_data
	- Linux: ~/.yuu_data

Commit:
Rewrite version progress 2:

- Move every parser to it's own version
- Delete parser.py, move the get_parser function to common.py
- Add better error handling.
- Add information about this branch

- Finalized GYAO, will do some testing later.
- Add ponimu.py (Contain some information)
- Add bilibili.py (Still empty)

marlenexxx commented:
what about this site?
i can give you a premium account for you use it for experiments.
https://www.kktv.me/browse/country/%E6%97%A5%E6%9C%AC

Commit:
More rewrite

noaione committed:
More and more rewrite and deletion
- Remove downloader.py and make it per-extension
- Add Aniplus support
- Add some workaround for different extension

noaione committed:
Fix typo

noaione committed:
Remove obsolete import

noaione committed:
Add missing return

Commit:
Add jsAES for Ponimu solution (Still doesn't work)

Commit:
Don't return if -R are omitted (Also fixed get_downloader())

Commit:
aniplus.py: return as normal text

Commit:
command.py: supress warn if resolution are 'best' or 'worst'

Commit:
Remove unnecessary comma

Commit:
command.py: add break if proxy check succeed

Commit:
ext: move url input to parser __init__

Commit:
Merge branch 'rewrite' of https://github.com/noaione/yuu into rewrite

Commit:
command.py: Use check_output, remove current updater

Commit:
ext.gyao.py: Fix most of the error

Commit:
ext.abematv.py: fix decryption

Commit:
command.py: pass input to extension __init__

Commit:
gyao.py: fix 'Response' object is not subscriptable

Commit:
Clean-up whitespaces

Commit:
command.py: don't pass input url to `parse` function
ext.abematv.py: shuffle chasePlayback

Commit:
Add a proper estimated filesize

Commit:
ext.abematv.py: move ?tver check to downloader class

Commit:
ext.gyao.py: remove additional return

Commit:
Fix No Encryption return

Commit:
ext.gyao.py: fix formatting

Commit:
Fix est size calculation
